velox/docs/configs.rst

```diff
@@ -354,3 +354,15 @@ Spark-specific Configuration
      - bool
      - true
      - If false, ``size`` function returns null for null input.
+   * - spark.runtime.bloom_filter.expected_num_items
+     - integer
+     - 1000000
+     - The default number of expected items for the runtime bloom filter in ``bloom_filter_agg`` function.
```

Comment:
Can we add a link to bloom_filter_agg's docs?

``:spark:func:`bloom_filter_agg```